### PR TITLE
Add guidelines for contributing feature requests, ideas, and other feedback

### PR DESCRIPTION
**Pull request by anawishnoff**
## Description

During the last WinUI Community Call, the team presented some general tips and guidelines on how our community can maximize their impact on WinUI through contributing requests, feedback, and ideas. This segment touched on the areas in which folks can have substantial impact, and how they can generally make their impact greater by focusing on actionable feedback and constructive phrasing.

We've expanded on this and turned it into a doc that can live on this repo. It outlines a bunch of sample areas where the community can have an impact on WinUI, and also gives lots of concrete examples of actionable feedback. I think it would be a good addition to live in our repo alongside the general [code contribution guidelines](https://github.com/microsoft/microsoft-ui-xaml/blob/master/CONTRIBUTING.md) and the [code of conduct](https://github.com/microsoft/microsoft-ui-xaml/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: CONTRIBUTING_feedback_and_requests.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Try this:
+  - Deprecation of past frameworks has left me burned. The following would go a long way in earning my trust because my company is trying to launch an app next year and will need it supported for at least 5 more: 
+    - OSS
+    - High-confidence 5-year roadmap
+    - Guaranteed support timeline
+
+    This feedback provides clear actionable items that the WinUI team can take into consideration and address. 
+    
+    
+## Sample areas where your feedback can have large impact
```
Comment (chingucoding) [inline CONTRIBUTING_feedback_and_requests.md:29]: What about mentioning the https://github.com/microsoft/microsoft-ui-xaml-specs repository in here to as part of feedback channel for new features? Adding feedback on new controls here definitely helps shape the API and create usable controls for everyone.

Comment (anawishnoff) [reply CONTRIBUTING_feedback_and_requests.md:29]: Great idea! I'm going to add this in under the features section.

File: CONTRIBUTING_feedback_and_requests.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+- ### Team Process
+  - Team process includes feedback about our repo and overall communication. Suggestions for improvement and proposed alternatives are very helpful here. 
+    - Ex: "`<`specific aspect of team process - e.g., response frequency, format`>`of the repo is making it hard for me to achieve `<`goal`>`. Please consider doing `<`alternative`>`instead because that would help..." 
+
+
+## Mutual respect
+
+We strive to be respectful & empathetic toward each other, and we extend this to our fantastic community as well.
+
+Our conduct guidelines help to ensure that discourse and feedback follows this same pattern of respect & empathy, and they also make it clear that non-construcive or hostile feedback is unacceptable. By following these guidelines, we can all enjoy the mutual respect that each WinUI team member, and each community member, deserve.
```
Comment (chingucoding) [inline CONTRIBUTING_feedback_and_requests.md:70]:
```suggestion
Our conduct guidelines help to ensure that discourse and feedback follows this same pattern of respect & empathy, and they also make it clear that non-constructive or hostile feedback is unacceptable. By following these guidelines, we can all enjoy the mutual respect that each WinUI team member, and each community member, deserve.
```

(fixed typo)

**Review by chingucoding — APPROVED** [review]
Looks good to me, great to see this being added! Should this be linked to somewhere (e.g. the issue template or the readme)?

**Review by SavoySchuler — APPROVED** [review]
Looks great, thanks for the awesome work here, @anawishnoff!

**Comment by anawishnoff** [comment]
@chingucoding Yes, great idea - I'm going to add links to some other docs once this one has been merged and we have a link for it :)

**Comment by chingucoding** [comment]
Great, looking forward to that @anawishnoff !

**Pull request merged**